Packs/CortexXDR/ReleaseNotes/3_0_13.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Added the *mitre_techniques_ids_and_names* and *mitre_tactics_ids_and_names* fields to the **xdr-get-incident-extra-data** command context outputs.
+- Breaking change - The argument severity in **xdr-insert-parsed-alert** command will now be required.
```

Comment:
can you say why? or add a default value to this arg?
```suggestion
- **Breaking change** - The argument severity in **xdr-insert-parsed-alert** command will now be required.
```

Reply:
The options for this arg are Informational, Low, Medium, High.
I don't think we want to have a default value here.
@JonathanMeler what do you think?

Reply:
If it will break backward compatibility, I suggest having a default one @Arsenikr can you recommend if there a clear candidate for the default value?

Reply:
I don't think we *can* define a default here between low/medium/high
there is no choice but to break BC here